dev_tools/utils/nxdl_utils.py

```diff
@@ -324,16 +299,10 @@ def get_own_nxdl_child(
     class_type - nxdl type or hdf classname (for groups, it is obligatory)
     hdf_name   - hdf name"""
     for child in nxdl_elem:
-        if "name" in child.attrib and child.attrib["name"] == name:
```

Comment:
RHS to be taken because the LHS construct is anyhow problematic:
if name is not in child.attrib referencing it will fail but maybe this failure will not trigger an exception, by contrast on the RHS the get call will trigger an exception and if it is thus the cleaner code